scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -16,260 +16,352 @@
-    echo
-    echo "Option 2: Skip FFmpeg installation."
-    echo
-
-    if yes_or_no_ffmpeg; then
-        if [[ -z $RPMFUSION_IS_INSTALLED ]]; then
-            yum install -y epel-release
-            rpm -Uvh https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
-        fi
-        yum install -y ffmpeg
-    else
-        echo "FFmpeg installation skipped. You may build FFmpeg from sources as described here: https://trac.ffmpeg.org/wiki/CompilationGuide/Centos"
-        echo
+    # Others
+    # TODO: cleanup and choose optional
+    pkgs+=(
+        # libusbx-devel
+        # gstreamer1
+        # gstreamer1-plugins-good
+        # gstreamer1-plugins-bad-free
+        # gcc
+        # gcc-c++
+        # make
+        # glibc-static
+        # glibc
+        # libstdc++-static
+        # libstdc++
+        # libstdc++
+        # libgcc
+        # cmake
+        # python36
+        # python36-pip
+        # glib2-devel
+        # flex
+        # gmp
+        # gsl
+        # libcap
+        # libcap
+        # gettext
+        # libXrandr
+        # libX11
+        # iso-codes
+        # mesa-libEGL
+        # mesa-libGLES
+        # mesa-libGL
+        # libgudev1
+        # libtheora
+        # cdparanoia
+        # pango
+        # mesa-libgbm
+        # alsa-lib
+        # libjpeg-turbo
+        # libvisual
+        # libXv
+        # opus
+        # libvorbis
+        # patch
+        # bzip2
+        # libv4l
+        # flac
+        # gdk-pixbuf2
+        # libdv
+        # mpg123
+        # libraw1394
+        # libavc1394
+        # libiec61883
+        # pulseaudio-libs
+        # libsoup
+        # speex
+        # wavpack
+        # boost-regex-1.53.0
+        # bluez-libs
+        # libusb
+        # libass
+        # libbs2b
+        # libchromaprint
+        # lcms2
+        # libssh2
+        # libdc1394
+        # libXext
+        # libssh
+        # libdca
+        # flite
+        # fluidsynth
+        # game-music-emu
+        # gsm
+        # nettle
+        # kate
+        # liblrdf
+        # libde265
+        # mjpegtools
+        # libmms
+        # libmodplug
+        # libmpcdec
+        # neon
+        # openal-soft
+        # OpenEXR
+        # openjpeg2
+        # openni
+        # libdvdnav
+        # librtmp
+        # librsvg2
+        # sbc
+        # libsndfile
+        # soundtouch
+        # spandsp
+        # libsrtp
+        # zvbi
+        # vo-amrwbenc
+        # webrtc-audio-processing
+        # wildmidi
+        # zbar
+        # libnice
+        # libxkbcommon
+        # opencore-amr
+        # libva
+        # python36-gobject
+        # python3-devel
+    )
+    [ -n "$optional" ] && pkgs+=()
+
+    if [ -n "$extra" ] ; then
+        # extra repository
+        extra_repos+=(
+            # https://mirrors.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
+        )
+        # TODO: cleanup
```

Comment:
is it ready to review PR? I why all packages are commented?

Reply:
It is ready for review in general, but I need some input from respective components regarding these packages.